src/AdminOnlyProperty/AdminOnlyPropertyConfigurationEditor.cs

```diff
@@ -14,10 +14,13 @@ internal sealed class AdminOnlyPropertyConfigurationEditor : ConfigurationEditor
         private readonly IDataTypeService _dataTypeService;
         private readonly PropertyEditorCollection _propertyEditors;
 
-        internal readonly string[] _defaultUserGroups = new[] { "admin" };
-        internal const string UserGroupsKey = "userGroups";
+        private readonly string[] _defaultUserGroups = new[] { "admin" };
```

Comment:
Also changed this from `internal` to `private` as it wasn't being referenced elsewhere in the codebase.